docs/infrastructure_and_maintenance/security/security_checklist.md

```diff
@@ -242,6 +234,76 @@ This header has several directives for fine-tuning the referrer information.
 - `Permissions-Policy` - limits what features the browser can use, such as fullscreen, notifications, location, camera, or microphone.
 For example, if someone succeeds in injecting their JavaScript into your site, this header prevents them from using those features to attack your users.
 
+### Disable weak cipher suites in TLS
+
+Consider blocking the use of TLS 1.2 and older versions.
+The newer TLS 1.3 does not include the weaker cipher suites that are included in 1.2 and older.
+Removing them means attackers can't attempt to force other users to use weak ciphers and eavesdrop on their communications.
+TLS 1.3 is as of December 2024 [supported by ca. 97% of global internet users](https://caniuse.com/tls1-3).
+If you need to support Internet Explorer or old versions of other browsers, you can disable TLS 1.1 and older, leaving 1.2 and 1.3 enabled.
+
+When using [[= product_name_cloud =]], you can [set the minimum TLS version in .platform/routes.yaml](https://docs.platform.sh/define-routes/https.html#enforce-tls-13).
+
+### Enable HTTP Strict Transport Security (HSTS)
```

Comment:
Or actually maybe I misunderstood it and it's ok to enforce HSTS, because the purge request is sent from the back office to Varnish, not the other way🤔 I'm not sure right now

Reply:
Nice catch. We can say that this should apply between the client and the public-facing proxy, if any, otherwise the webserver. I'll come up with a sentence or warning box about it.

Reply:
It's safer for me to ping @vidarl here, he knows better what's the recommended approach 😄

Reply:
These settings are related to what happends between client and public-facing proxy, yes.

Even with both https and HSTS enabled, varnish and app container will still be reachable to each other over plain http

Reply:
Thanks, handled in https://github.com/ibexa/documentation-developer/pull/2565/commits/7eb8d61d5482fdcf3f1dafc3c0534b4a847bc230